src/js/outreach/views/opt-in_views.js

```diff
@@ -98,10 +98,10 @@ const OptInView = View.extend({
   setSubmitButtonState() {
     const firstName = this.ui.firstName.val();
     const lastName = this.ui.lastName.val();
-    const dob = this.ui.dob.val();
+    const birthDate = this.ui.birthDate.val();
     const phone = this.ui.phone.val();
```

Comment:
This actually might be a little broken.. if `parsePhoneNumber` fails this will be true, but `phone` in the state model would be `null`.  But that's probably and edge case for now and will be fixed by doing the event driven state changes.